main.go

```diff
@@ -120,12 +123,37 @@ func sendEmail(event *types.Event) error {
 		return bodyErr
 	}
 
-	msg := []byte("To: " + toEmail + "\r\n" +
+	msg := "To: " + toEmail + "\r\n" +
 		"Subject: " + subject + "\r\n" +
 		"\r\n" +
-		body + "\r\n")
-
+		body + "\r\n"
+
+	if insecure {
+		if smtpconn, connErr := smtp.Dial(smtpAddress); connErr != nil {
+			return connErr
+		}
+		if connErr != nil {
+			return connErr
+		}
+		defer smtpconn.Close()
+		smtpconn.Mail(fromEmail)
+		smtpconn.Rcpt(toEmail)
+		if smtpdata, dataErr := smtpconn.Data(); dataErr != nil {
+			return dataErr
+		}
+		if dataErr != nil {
+			return dataErr
+		}
+		defer smtpdata.Close()
+		buf := bytes.NewBufferString(msg)
+		if _, dataErr = buf.WriteTo(smtpdata); dataErr != nil {
+			return dataErr
+		}
+
+		return nil
+	}
 	return smtp.SendMail(smtpAddress, smtp.PlainAuth("", smtpUsername, smtpPassword, smtpHost), fromEmail, []string{toEmail}, msg)
```

Comment:
It looks like `SendMail` expects `[]byte` rather than a string, so you could do the following:
```suggestion
	return smtp.SendMail(smtpAddress, smtp.PlainAuth("", smtpUsername, smtpPassword, smtpHost), fromEmail, []string{toEmail}, []byte(msg))
```

OR

Add the byte array back `msg := []byte("To: " + toEmail + "\r\n" +` on line 26 AND change `NewBufferString` to `buf := bytes.NewBuffer(msg)` on line 148.